nixos/doc/manual/configuration/x-windows.chapter.md

```diff
@@ -116,48 +116,37 @@ using lightdm for a user `alice`:
 
 ## Intel Graphics drivers {#sec-x11--graphics-cards-intel}
 
-There are two choices for Intel Graphics drivers in X.org: `modesetting`
-(included in the xorg-server itself) and `intel` (provided by the
-package xf86-video-intel).
-
-The default and recommended is `modesetting`. It is a generic driver
-which uses the kernel [mode
-setting](https://en.wikipedia.org/wiki/Mode_setting) (KMS) mechanism. It
+The default and recommended driver for Intel Graphics in X.org is `modesetting`
+(included in the xorg-server package itself).
+This is a generic driver which uses the kernel [mode
+setting](https://en.wikipedia.org/wiki/Mode_setting) (KMS) mechanism, it
 supports Glamor (2D graphics acceleration via OpenGL) and is actively
-maintained but may perform worse in some cases (like in old chipsets).
-
-The second driver, `intel`, is specific to Intel GPUs, but not
-recommended by most distributions: it lacks several modern features (for
-example, it doesn't support Glamor) and the package hasn't been
-officially updated since 2015.
-
-The results vary depending on the hardware, so you may have to try both
-drivers. Use the option
-[](#opt-services.xserver.videoDrivers)
-to set one. The recommended configuration for modern systems is:
+maintained, however it still lacks Vsync and may perform worse in some cases
+(like in old chipsets).
 
-```nix
-{
-  services.xserver.videoDrivers = [ "modesetting" ];
-}
-```
+::: {.note}
+The `modesetting` driver doesn't currently provide Vsync (this will become
+available in an upcoming X.org release), So, without using a compositor (for
+example, see [](#opt-services.picom.enable)) you will experience screen
+tearing.
+:::
 
-If you experience screen tearing no matter what, this configuration was
-reported to resolve the issue:
+There also used to be a second driver, `intel` (provided by the
```

Comment:
Can we just drop this entire thing? It's really not worth it.

Reply:
No, you already dismissed in https://github.com/NixOS/nixpkgs/issues/342763, left it there for 3 months and the result is that I've found at least three people with a broken desktop in the forum. It needs to be documented.

Reply:
I mean drop the entire DDX, and force people to switch off of it.

Reply:
Removing `videoDrivers`? It's too late for the current release, but it can be done for 25.05. I would still wait for the "recent" changes to `modesetting` to reach a stable X.org release (or building xorg-server from master), because to me the current driver still looks like a downgrade compared to other like `amdgpu`.

Reply:
We can't remove `videoDrivers` because Nvidia, but we can remove the Intel DDX and the option value (and probably same for amdgpu, which is also very not recommended).

Reply:
Ok, It makes sense to remove `xf86-video-intel` if it's broken.